test/xskbench/xskbench.c

```diff
@@ -563,24 +578,25 @@ SetupSock(
     AttachXdpProgram(Queue);
 }
 
+_Kernel_float_used_
```

Comment:
I've never seen this before. What does it do?

Reply:
It's related to https://learn.microsoft.com/en-us/windows-hardware/drivers/devtest/28110-floating-point-hardware-protect. In Windows kernel, there are certain APIs you need to use in order to use floating point operations in a reliable way.